docs/references/rest-apis/rest-session-api.md

```diff
@@ -301,6 +289,11 @@ An object containing information about the enabled authentication methods.
           A port that can be used for certificate based authentication.
 
   
+  * **message**: `string` (**optional**)
+      Reports the content of the Login Banner, if configured on the device. A browser based application should display this message to the user before login if this property is set.
+          This property will be missing if the login banner is not enabled.
+
+  
```

Comment:
Maybe just add an "Example responses:" title to make it clear what the reported JSON mean

```suggestion
 Example responses:
```